tests/baselines/reference/privateNameBadAssignment.errors.txt

```diff
@@ -0,0 +1,30 @@
+tests/cases/conformance/classes/members/privateNames/privateNameBadAssignment.ts(1,1): error TS2304: Cannot find name 'exports'.
+tests/cases/conformance/classes/members/privateNames/privateNameBadAssignment.ts(6,3): error TS2339: Property '#foo' does not exist on type 'typeof B'.
+tests/cases/conformance/classes/members/privateNames/privateNameBadAssignment.ts(11,9): error TS2304: Cannot find name 'exports'.
+tests/cases/conformance/classes/members/privateNames/privateNameBadAssignment.ts(12,14): error TS2339: Property '#foo' does not exist on type 'C'.
+
+
+==== tests/cases/conformance/classes/members/privateNames/privateNameBadAssignment.ts (4 errors) ====
+    exports.#nope = 1;           // Error (outside class body)
+    ~~~~~~~
+!!! error TS2304: Cannot find name 'exports'.

```

Comment:
Note: I found the JS  version of this test below, so disregard that part. And since `exports` isn't resolved, no error is expected on `#nope` or `#bar`. And it turns out in TS that `A.prototype` is defined for functions, with type `any`, so `#no` shouldn't have an error either.